### DISCUSSION TITLE
Multiple Tables in a View

### DISCUSSION THREAD

**Discussion by LeeMangold** [Laravel-Backpack/community-forum#465] [Q&A (Help)] [answered]
I would like to show multiple tables in a view. I really like the way backpack handles list views, and I was wondering if there's any way to either (1) show multiple tables in the ListView or (2) Reuse the table mechanism in my own views somehow.

Thanks!

**Comment by pxpm** [comment, marked as answer]
@LeeMangold thanks for the question.

I have both good and not so good news.

It's doable, but it's not the easiest of the jobs. Some people already created some addons to help with that, for example https://github.com/izica/relations-widgets-for-backpack (even if it's not what you want, you can get the idea of what needs to be done to have an extra table). 

The good news is that we plan to add support for this in core, something we want to work out after we launch v6 https://github.com/Laravel-Backpack/community-forum/issues/100#issuecomment-1465660398

Sorry I can't be helpful at the moment 🙏 

Cheers
